doc/api/process.md

```diff
@@ -1247,6 +1247,22 @@ process.kill(process.pid, 'SIGHUP');
 *Note*: When `SIGUSR1` is received by a Node.js process, Node.js will start
 the debugger, see [Signal Events][].
 
+## process.libc
+<!-- YAML
+added:
+-->
+
+* {string}
+
+The `process.libc` property returns the C standard library used to compile
+the Node.js binary. For instance `glibc` or `musl`.
+
+<!-- eslint-disable semi -->
+```js
+console.log(`This binary was compiled using ${process.libc}`);
+```
```

Comment:
I'm wondering if it would be better to expose this via `process.versions.libc`

Reply:
it would stand out as not actually being a version unless it was like `glibc-2.18` and `musl-1.1.18` and then it isn't really usable for the intended purpose. we could add both the above format to versions and just the name `glibc` or `musl` or whatever else to config

Reply:
Or we could do:

```
process.versions.lib = {
    name: "glibc",
    version: "2.18"
}

```

Reply:
What about adding it as it is?

```js
process.versions.glibc = '2.18.0'
// or
process.versions.musl = '1.1.18'
```

That way it can easily be checked what is used and in what version and there is no "special" handling necessary.